include/bluetooth/bluetooth.h

```diff
@@ -271,6 +271,12 @@ enum {
 
 	/* Advertise using GAP device name */
 	BT_LE_ADV_OPT_USE_NAME = BIT(3),
+
+	/* Use low duty directed advertising mode */
+	BT_LE_ADV_OPT_DIR_MODE_LOW_DUTY = BIT(4),
+
+	/* Use high duty directed advertising mode */
+	BT_LE_ADV_OPT_DIR_MODE = BIT(5),
```

Comment:
We only have 8 bits to spare for the options, so I think it'd be better to use  `peer_addr != NULL` to know we're doing directed and then a single flag to toggle between low or high duty.